h/api/test/views_test.py

```diff
@@ -65,7 +66,7 @@ def user(monkeypatch):
     return user
 
 
-def test_index():
+def test_index(replace_io):
```

Comment:
I think if you're not using it in the function it would be cleaner to use the decorator:

```python
@pytest.mark.usefixtures('replace_io')
```